app/controllers/partner_requests_controller.rb

```diff
@@ -1,13 +1,13 @@
 class PartnerRequestsController < ApplicationController
   before_action :authenticate_user!
+  before_action :verify_status_in_diaper_base
```

Comment:
Does this cause a redirect when accessing the index page? If so, I think we don't want to do that. Because the partners should still have access to historical requests.

Reply:
In addition, would you be able to add text in the area where the request buttons would be? Such that we can explain to the users why the buttons are not visible?

Reply:
This action permit the Partner access the historical requests normally, and this doesn't redirect to index page.
About the text, this action shows a message that the partner has been deactivated but we can add text in the request button area

Reply:
hey there @EmersonManabuAraki -- I was wondering about:

> This action permit the Partner access the historical requests normally, and this doesn't redirect to index page.

The definition of `verify_status_in_diaper_base` is:
```
  def verify_status_in_diaper_base
    if current_partner.status_in_diaper_base == "deactivated"
      flash[:alert] = 'Your account has been disabled, contact the organization via their email to reactivate'
      redirect_to partner_requests_path
    end
  end
```

Wouldn't this prevent access to the #index action if `current_partner.status_in_diaper_base == "deactivated"` is true?